src/client/components/AddResume/AddResume.js

```diff
@@ -22,20 +23,24 @@ export const AddResume = () => {
   const handleChange = (e) => {
     const selectedFile = e.target.files[0];
 
-    setUploadedFile(selectedFile);
+    setUploadedMyFile(selectedFile);
 
     if (selectedFile) {
       if (types.includes(selectedFile.type)) {
         setError(null);
-        setUploadedFile(selectedFile);
+        setUploadedMyFile(selectedFile);
       } else {
-        setUploadedFile(null);
+        setUploadedMyFile(null);
         setError('Please select a pdf file');
       }
     }
   };
 
-  useStorage(uploadedFile);
+  const {url} = useStorage(uploadedMyFile);
+
+  const clickHandler = () => {    
```

Comment:
Close, but not quite. I think your reasoning is to think in props - that is one way of doing things, but there are better ways, and how you are thinking in hooks. 

That means we avoid passing props, and instead have shared "states". Here is an illustration, where you need to swap "store" with "my-hook". 

![image](https://user-images.githubusercontent.com/8851125/108971055-62c78100-7682-11eb-9ac2-e76f4533ee96.png)

in other words, use the useStorage in posturlcomponent.